src/routes/basic-form-success.tsx

```diff
@@ -0,0 +1,29 @@
+import { GoAButton, GoACallout } from '@abgov/react-components';
+import { useNavigate } from 'react-router-dom';
+
+export function BasicFormSuccessRoute() {
+  const navigate = useNavigate();
+  return (
+    <main>
+      <div className="space-5" />
+
+      <GoACallout type="success">
+        Your form has been submitted successfully!
+      </GoACallout>
+
+      <h2>Thanks for trying out the product template</h2>
+      <h3>
+        You can use this project as a starting point in your service. 
+        <a href="https://github.com/GovAlta/ui-components">Find the code in Github.</a>
+      </h3>
+      <h3>
+        Go back to the home page to try out and see 
+        <a href="https://ui-components-ui-components-dev.os99.gov.ab.ca">more available components,</a> <a href="https://github.com/GovAlta/ui-components/issues">give feedback</a>, and <a href="https://www.npmjs.com/package/@abgov/react-components">start using the design system.</a>
```

Comment:
I think we should maybe link to the web-components instead of the react-components for the get started link

Reply:
but this the react template app

Reply:
Touche